targets/simple_switch/sample_extern/macsec/Makefile

```diff
@@ -0,0 +1,11 @@
+crypto_SOURCES = crypto.cpp
+crypto_LDADD = -lcrypto
+
+CPPFLAGS += -I../../ -fPIC -shared
+
+libexterncrypto.so : $(crypto_SOURCES)
+	g++ $(CPPFLAGS) $(crypto_SOURCES) -o libexterncrypto.so $(crypto_LDADD)
+
+clean:
+	rm -rf libexterncrypto.so
+
```

Comment:
Since this extern is a bit complicated, I think we should have a simple C++ test and a `make check` target in this Makefile to make sure that the extern definition is not broken and that encryption / decryption work properly.

Reply:
Let me think about this one and get back.